Structure/Structure/Class1.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+
+namespace Structure
+{
+    public struct ZPower
+    {
+        //Определение полей
+        public double Base { get; set; }
+        public int Exponent { get; set; }
+        //
+        public double Value
+        {
+            get
+            {
+                return Math.Pow(Base, Exponent);
+            }
+        }
+
+        public bool IsPositive
+        {
+            get
+            {
+                return Value > 0;
+            }
+        }
```

Comment:
Можно проще, меньше инфраструктурного кода
```suggestion
        public double Value  => Math.Pow(Base, Exponent);

        public bool IsPositive => Value > 0;
```